overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -19473,7 +19478,7 @@
           "autoclaim:1": 0,
           "ismain:1": 1,
           "name:8": "Assembly Line",
-          "desc:8": "Not a project for the faint of heart.\n\nMake sure your power and circuit production are up to the challenge before you try to build this.\n\nNotice that all numbers of required items are multiples of 12. This is to make an Assembly Line that\u0027s 12 units long, allowing for 12 inputs. 12 is enough to make a Fusion Reactor Controller Mk1, but you\u0027ll need an even longer Line for later projects.\n\nThe Fluid Input Hatches can replace any of the Steel Casings along the bottom outside except for the Casings touching the Output Bus. The Energy Input Hatches along the top can be replaced by Steel Casings, and you only need the one Energy Input Hatch (although you can add more if you want). The Input Buses along the bottom must be ULV tier, nothing else."
+          "desc:8": "The §3Assembly Line§r is a large multiblock structure that crafts complex materials, similar to an §3Assembling Machine§r. These will be crucial going forward for making a variety of components and machinery.\n\nMake sure your power generation and circuit production are up to the challenge. It is useful to make at least two of these to avoid recipe conflicts and for different programmed circuits.\n\nNotice that all numbers of required items are §emultiples of 12§r. This is to make an Assembly Line that\u0027s 12 units long, allowing for 12 inputs. 12 is enough to make a §aFusion Reactor Controller Mk1§r, but you\u0027ll need an even longer Assembly Line for later projects.\n\n§aFluid Input Hatches§r can replace any of the §6Steel Casings§r along the bottom outside except for the Casings touching the §aOutput Bus§r. For the most part, ULV ones work fine, but a few recipes will need an LV one.\n\nThe §aEnergy Input Hatches§r along the top can be replaced by Steel Casings: you only need the one Energy Input Hatch (although you can add more if you want).\n\nThe §aInput Buses§r along the bottom §emust be ULV tier§r, nothing else. Make sure to spin them all so the hole is on the bottom."
```

Comment:
12 units long, allowing for 11 inputs.

Reply:
Yeah it was in the original text and I didn't catch it. I've changed it to distinguish that the length is 12 but it has 11 inputs.